### ISSUE TITLE
Robots becoming invisible

### ISSUE THREAD

**Issue by Amelia-** [BuildCraft/BuildCraft#2620]
After I updated from Buildcraft 6.4.3 to 6.4.8 my robots seem to turn invisible.
At first I thought they were getting lost and such again because the docking stations were staying red but no robot to be found nearby.
Then I placed a robot on a green docking station and the docking station turned red but there was no visible robot.
After that I started watching them a bit and I noticed trees getting cut down by invisible things which turned out to be my missing lumberjack robots.
I went a few chunks away and came back and one of the lumberjacks and pickers became visible again.

**Comment by hea3ven** [comment]
Is this on a server or single player?, also, what other mods are you using?

**Comment by Amelia-** [comment]
It's on a server.
I'll need to make a list of mods, do you need to know all of them?
I've not seen other entities going invisible, I have quite a few Railcraft trains sitting around and they are fine.
Fastcraft does some things with rendering right? I can try disabling that and see if that helps.
It also seems to affect all of the robots I am using for this, pickers, lumberjacks and planters.

**Comment by hea3ven** [comment]
If you are using a modpack you can just say which one it is.

Are the robots invisible only after placing them, or do they randomly go invisible?

**Comment by Amelia-** [comment]
It's a private modpack I put together so I would need to make a list.
However disabling Fastcraft on the client seemed to have fixed the problem. I played for a few hours last night with it off and watched the robots at times and not once did they go invisible.
I guess it is an optimisation that Fastcraft is doing with the rendering?

The robots get invisible at random moments and stay invisible for long periods. But one I placed was invisible right as I placed it.
Usually 1 or 2 out of the 7 robots I had there would be visible after a while, the rest invisible.
